networks/suzuka/suzuka-client/Cargo.toml

```diff
@@ -12,13 +12,24 @@ rust-version = { workspace = true }
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+async-channel = { workspace = true }
```

Comment:
What's `async-channel` for?

Reply:
And Old thing I was doing, left over. Removed now :)